scripts/deploy_subnet_under_calibration_net/prepare_local.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# A script which resets your local .ipc folder so we can run the deploy.sh script for a local deployment
+#
+# To run:
+# 1. Run `./prepare_local.sh`
+# 2. Run `./deploy.sh`
+#
+# Known issues:
+# - The deploy.sh script does not run "make docker-build" which can cause errors if its outdated on your
+#    machine. To fix this, run `make docker-build` in ipc/fendermint folder and rerun the deploy.sh script
+
+set -eo pipefail
+
+SCRIPT_DIR=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+echo "SCRIPT_DIR: $SCRIPT_DIR"
+
+echo "Installing dependencies..."
+cargo install toml-cli
+
+echo "Removing existing .ipc folder"
+rm -rf ~/.ipc
```

Comment:
This seems a bit brutal. Should it perhaps make a backup, if it doesn't exist yet?

Reply:
> This seems a bit brutal. Should it perhaps make a backup, if it doesn't exist yet?

Maybe call it `reset_local_.sh` and/or add a prompt to confirm to user that it will reset your ipc folder?

Reply:
The upsert trick from @aakoshh would work for the ipc-cli, but this folder is also storing validator config and keys. @fridrik01 alternatively create a timestamped folder?